src/main/java/thrift/storage/JsonAdaptedTransaction.java

```diff
@@ -36,8 +36,8 @@
      */
     @JsonCreator
     public JsonAdaptedTransaction(@JsonProperty("type") String type, @JsonProperty("description") String description,
-                                  @JsonProperty("value") String value, @JsonProperty("date") String date,
-                                  @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
+            @JsonProperty("value") String value, @JsonProperty("date") String date,
```

Comment:
Does checkStyle complain that newline indents aren't 8 spaces? If not, ignore this comment.

Reply:
There are 8 spaces.